RijsMuseum/app/src/main/java/com/example/rijsmuseum/fragments/ArtifactsFragment.kt

```diff
@@ -58,7 +58,7 @@ class ArtifactsFragment : Fragment() {
     private fun getAdapter() {
         adapter = RecyclerAdapter()
         adapter.setOnItemClickListener(object : RecyclerAdapter.OnClickCallback {
-            override fun onItemClick(cList: Collections.ArtObject) {
+            override fun onItemClick(cList: CollectionsResource.ArtObjectResource) {
```

Comment:
Again here. The resource models should be used only in the network module. In the presentation layer you should use your domain models.